src/lib/DataTable.tsx

```diff
@@ -259,16 +278,34 @@ class TableHead<T> extends React.Component<TableHeadProps<T>> {
                 data-index={index}
                 colSpan={colSpan}
                 rowSpan={rowSpan}
-                style={column.customStyle && column.customStyle({header: true, name: column.name})}
+                style={style}
                 onClick={this._getOnSortClick(column)}
             >
                 <div className={b('head-cell')}>
                     {header}
                     {<ColumnSortIcon {...column} />}
                 </div>
+                {this.renderCellResizeHandler(column)}
```

Comment:
maybe use same approach as for `ColumnSortIcon`?
`<ResizeHandler {...column} onResize={onResize} />`

Reply:
I wanted to check whether it's needed and return null separately, so there won't be additional logic inside `ResizeHandler` itself

Reply:
you can add conditional render here. in method you rewrite propnames, write own type instead of using column's type and method are usually used in class components approach, not functional. not a mistake, just a bit awkward